nextcord/application_command.py

```diff
@@ -694,12 +694,42 @@ def from_callback(
             if option_class:
                 first_arg = True
                 typehints = typing.get_type_hints(self.callback)
+                callback_params = signature(self.callback).parameters
+                possible_options: Set[str] = set()
+
+                for i, param in enumerate(callback_params.values()):
+                    if (
+                        param.annotation is param.empty
+                        or param.annotation is Interaction
+                        or issubclass(param.annotation, Interaction)
+                    ) and i in (0, 1):
+                        # self or interaction parameter, ignore
+                        continue
```

Comment:
Why the `continue`, negate the condition and move the actual loop body into the condition body?